tests/contracts/EventDispatcher/Event/SuggestionEventTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Contracts\Search\EventDispatcher\Event;
+
+use Ibexa\Contracts\Search\Event\BuildSuggestionCollectionEvent;
+use Ibexa\Search\Model\SuggestionQuery;
+use PHPUnit\Framework\TestCase;
+
+final class SuggestionEventTest extends TestCase
```

Comment:
It seems that the event was named `SuggestionEvent` in the past as the PR description and this test class name suggest. If `BuildSuggestionCollectionEvent` is the definitive name for this event, the test class name should also be updated.
```suggestion
final class BuildSuggestionCollectionEventTest extends TestCase
```
Mabe there is some other `SuggestionEvent` related names that should be updated to `BuildSuggestionCollectionEvent`, I didn't deep dive into this.